Assets/MixedRealityToolkit/_Core/Definitions/Devices/InteractionMapping.cs

```diff
@@ -244,4 +243,129 @@ public void SetValue(Tuple<Vector3, Quaternion> newValue)
 
         #endregion Set Operators
     }
+
+    public class Test
+    {
+        System.Collections.Generic.List<IInteractionMapping> list = new System.Collections.Generic.List<IInteractionMapping>();
+        void helloWorld()
+        {
+            var interaction = new InteractionMapping<float>(0, AxisType.SingleAxis, DeviceInputType.None, new InputAction(1, "None"));
```

Comment:
I REALLY like the idea of typing the interaction mapping. much easier to read and consume

Reply:
Ideally we shouldn't be making new ones anyway. They should be done in the controller's config profile.